src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+// default state
+const defaultState = {
+  books: [
+    { Id: '01', Title: 'Why Nations fail', Author: 'James' },
+    { Id: '02', Title: 'Book2', Author: 'Author 2' },
+  ],
+};
+
+const ADD_BOOKS = 'bookstore/books/ADD_BOOKS';
+const REMOVE_BOOKS = 'bookstore/books/REMOVE_BOOKS';
+
+// actions
+export const addBook = (newBook) => ({
+  type: ADD_BOOKS,
+  newBook,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOKS,
+  id,
+});
+
+// reducer
+
+const bookReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case ADD_BOOKS:
+      return {
+        books: [...state.books, action.newBook],
+      };
+    case REMOVE_BOOKS:
+      return {
+        books: [...state.books.filter((book) => book.Id !== action.id)],
+      };
+    default:
+      return state;
+  }
+};
+
+export default bookReducer;
```

Comment:
- [ ] Hi @habasm, once more, let's kindly adhere to the requirements of this project. Let's kindly import the necessary methods from Redux Toolkit as requested and define slice of state for books and categories. **Hints** `import { createSlice } from "@reduxjs/toolkit"` Next, you'll kindly Create an initialState with a state value books that contains an empty array. **Hints**
```
initialState: {
    books: [],
  }
```

- After creating two reducers addBook and removeBook export by default bookReducer.reducer. Reducers should be created inside:

```
export const bookReducer = createSlice({

})
```

- Kindly check the requirements once more below on the screenshot:

![image](https://user-images.githubusercontent.com/63087491/221787239-77526f62-5e28-4727-8f80-d85bc10595a9.png)